### ISSUE TITLE
How to implement weight decay towards the pre-trained model?

### ISSUE THREAD

**Issue by sedol1339** [OptimalScale/LMFlow#906]
Hello, let me one question.

If using LMFlow for supervised fune-tuning, how do I implement penalizing the distance between starting and current weights? This was shown to be effective in https://arxiv.org/abs/1909.11299
